### DISCUSSION TITLE
The database you introspected seems to belong to a Prisma 1 project

### DISCUSSION THREAD

**Discussion by aesth0x** [prisma/prisma#4044] [Q&A] [answered]
I did the following to initialize the project:
```
npm install -D @prisma/cli
npx prisma init
```
When running:
```
npx prisma introspect
```
I saw the prompt:
```
┌────────────────────────────────────────────────────────────────────────┐
│                                                                        │
│  Upgrading from Prisma 1 to Prisma 2                                   │
│                                                                        │
│  The database you introspected seems to belong to a Prisma 1 project.  │
│                                                                        │
│  Please run the following command to upgrade to Prisma 2.0:            │
│  npx prisma-upgrade [path-to-prisma-yml] [path-to-schema-prisma]       │
│                                                                        │
│  Note: `prisma.yml` and `schema.prisma` paths are optional.            │
│                                                                        │
│  Learn more about the upgrade process in the docs:                     │
│  https://pris.ly/d/upgrading-to-prisma2                                │
│                                                                        │
└────────────────────────────────────────────────────────────────────────┘
```
Question 1: This is a newly created postgresql database, and I don't know why it belongs to Prisma 1.
Qeustion 2: I don't have a prisma.yml in my directories, which prevents me from upgrading from Prisma 1 to Prisma 2. What's the workaround?

Thanks in advance!

**Comment by ryands17** [comment, marked as answer]
Hey @lys1030 👋 
This is just an incorrect message and will not affect any functionality. It would be great if you share the SQL with us so that we can reproduce this and fix it :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
